### DISCUSSION TITLE
Discussion about the Mesa Vision, Strategy and Mission

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#2235] [Future of Mesa]
Thanks to @EwoutH for starting these! 

## Background:

I have now been in several organizations in my lifetime, but at one time was in an organization that lacked these core documents. Previously, I would have ignored these documents or thought them irrelevant, but what that experience taught me, watching Mesa development debates, and gaining a deeper understanding of learning theories is that these documents provide the context or scaffolding to inform decisions. To put it in my own complexity metaphor, they provide the core dynamics of the ecosystem which allows the different species to grow interdependently. 

To that end below are my current thoughts to get the discussion going. 

## Vision: 

An open source simulation ecosystem that allows for rapid development of valid models to help humanity develop deeper understanding and make better decisions from the individual to the global level.

## Strategy: 
The Mesa strategy has three goals: 

**Accessible:** A simulation ecosystem that allows for simulations from beginner, simple models to advanced highly complex models. A welcoming, positive, community that invites and develops contributions from the new to the experienced.  

**Resilient:** A library that is not dependent on any one person but it is a robust and global community

**Innovative:**  A thought leader in the simulation field that is creating new ways to leverage computational for the benefit of humanity and the Earth. 

#TODO: Develop clear metrics and tangible sub-goals to reach these overarching goals.  

## Mission: 

To provide an open-source simulation ecosystem that is accessible from the novice to the expert that improves understanding and decision making for good.

**Comment by EwoutH** [comment]
Thanks for starting this one. I think one thing we have to do is get a very clear picture of our target audience. Which groups are they, what are the priorities and requirements of each and how big is each group. And which groups might be able to deliver back to us, and if so, what.
